ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -38,15 +38,30 @@
                   <p>Ensure ALL of the information below is correct before making any changes to this
                     registration. Necessary fees will be applied as updates are made.
                   </p>
+
+                  <!-- Unit Note Info -->
                   <p v-if="getMhrUnitNotes && getMhrUnitNotes.length >= 1">
                     There are unit notes attached to this manufactured home.
-                    <span v-if="!isRoleStaffReg">
-                      To view unit note information on this home, complete a manufactured home search.
+                    <span v-if="isRoleStaffReg">
+                      <a href="#unit-note-component">See Unit Notes</a>
                     </span>
                     <span v-else>
-                      <a href="#unit-note-component">See Unit Notes</a>
+                      To view unit note information on this home, complete a manufactured home search.
                     </span>
                   </p>
+
+                  <!-- Has Caution Message -->
+                  <template v-if="getMhrInformation.hasCaution">
+                    <header id="yellow-message-bar" class="message-bar mt-9">
```

Comment:
This is a good candidate for a common component.

Reply:
Definitely! I thought i've done this, but previous project.

We looked at some designs today that had another variation, styling and messaging. 
I think once we implement that, would be a good time to do so.

Reply:
Does caution box component not work for this one?

Reply:
oh, i'm sure it would!

Would require some changes, for both this instance and the designs we looked at today.